administrator/language/en-GB/com_guidedtours.ini

```
@@ -66,6 +66,16 @@ COM_GUIDEDTOURS_TITLE="Title"
 COM_GUIDEDTOURS_TITLE_TRANSLATION="Title (%s)"
 COM_GUIDEDTOURS_TOURS_DUPLICATED="%d tours duplicated."
 COM_GUIDEDTOURS_TOURS_DUPLICATED_1="Tour duplicated."
+COM_GUIDEDTOURS_TOURS_IMPORTED="%d tours imported successfully."
```

```suggestion
COM_GUIDEDTOURS_TOURS_IMPORTED="%d tours imported."
```

Remove redundant word